client/prisma/schema.prisma

```diff
@@ -34,3 +34,13 @@ model Listener {
 
     @@map("listener")
 }
+
+model Lockup {
+  id       Int @default(autoincrement()) @id
+  address  String @unique
+  amount   String
+  end      String
+  weeks    Int @default(0)
```

Comment:
There should always be a `weeks` value present when creating a Lockup object I think? If I am not mistaking I would remove the default value and rather have it fail with an error when no `weeks` present at object creation.

Reply:
Agreed, updated.